pull-request.py

```diff
@@ -268,14 +269,17 @@ def add_reviewers(entry, reviewers, team_reviewers):
 API_VERSION = "v3"
 BASE = "https://api.github.com"
 
+pr_token = os.environ["PULL_REQUEST_TOKEN"] if "PULL_REQUEST_TOKEN" in os.environ else get_envar("GITHUB_TOKEN")
```

Comment:
Just taking a quick look before the weekend - this does look reasonable, but could you please update:

1. the README to include all new variables and descriptions
2. add a full example to the examples folder of doing a pr for a different repository

And these lines can be simplified to be:

```suggestion
PR_TOKEN = os.environ.get("PULL_REQUEST_TOKEN") or get_envar("GITHUB_TOKEN")
```
And I made all caps to be consistent with other variables around it.

Reply:
thanks, fixed